logger.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 var pino = require('pino')
+var id = Symbol.for('id')
```

Comment:
should this be Symbol.for('pino-http') - there's a fair chance in the same process someone could do Symbol.for('id') and possibly rely on Symbol.keyFor(sym)